test/distributed/test_symmetric_memory.py

```diff
@@ -133,7 +135,7 @@ def test_cuda_nvlink_connectivity_detection(self) -> None:
         for row in connectivity.matrix:
             self.assertEqual(len(row), torch.cuda.device_count())
 
-    @skipIfRocm
+    @runOnRocmArch(MI300_ARCH)
```

Comment:
I think `SymmetricMemoryTest` doesn't run on CI yet due to lack of hosts with P2P support. Can you please manually verify for now if these tests pass?